# Request
float64 vx
float64 vy
float64 ax
float64 ay
float64 obs_dist_abv
float64 obs_dist_blw
float64 heading_dist
float64 progress
float64 h_val_abv
float64 h_val_blw
float64 alpha_abv
float64 alpha_blw
bool solver_status
---
# Response
bool success
float64[] alpha_dot

